### DISCUSSION TITLE
DB name is not added to url with replica set

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [typegoose/mongodb-memory-server#613] [Q&A] [answered]
Hello,

I have this replica set configuration:
```
this.replSet = await MongoMemoryReplSet.create({
                    replSet: {
                        count: 1,
                        dbName: 'my-db',
                    },
                });
                await this.replSet.waitUntilRunning();
```

and i get uri with:
```
const mongoUri = this.replSet.getUri()
```

output is: `mongodb://127.0.0.1:52088/?replicaSet=testset`
expected output: `mongodb://127.0.0.1:52088/my-db?replicaSet=testset`

**Comment by hasezoey** [comment, marked as answer]
this is currently not enabled by default, see comment tsdoc:
https://github.com/nodkz/mongodb-memory-server/blob/1154d10d861077e143125aaa401ac269d556c1cb/packages/mongodb-memory-server-core/src/MongoMemoryReplSet.ts#L304

TL;DR: by default no database is added to the URI, because of different parser implementations
